packages/core/.gitignore

```diff
@@ -1,5 +1,7 @@
 cache/
+cache-ovm/
 artifacts/
+artifacts-ovm/
```

Comment:
Contracts are built to new folders when compiled with `--network optimism`, which is probably why `getAbi`, `artifacts.require(...)`, and `.deployed()` all fail right now when testing a script on `--network optimism`